app/pytorch_load.ipynb

```diff
@@ -0,0 +1,168 @@
+    "    \"\"\"\n",
+    "\n",
+    "    def __init__(self, path: str):\n",
+    "        \"\"\"\n",
+    "        If there's a GPU available, tell PyTorch to use the GPU.\n",
+    "        Loads model and tokenizer from saved model directory (path)\n",
+    "        \"\"\"\n",
+    "        if torch.cuda.is_available():\n",
+    "            self.device = torch.device('cuda')\n",
+    "        else:\n",
+    "            self.device = torch.device('cpu')\n",
+    "        self.model = BertForSequenceClassification.from_pretrained(path)\n",
+    "        self.tokenizer = BertTokenizer.from_pretrained(path)\n",
+    "        self.model.to(self.device)\n",
+    "\n",
+    "    def predict(self, text: str):\n",
+    "        \"\"\"\n",
+    "        Makes a binary classification prediction based on saved model\n",
+    "        \"\"\"\n",
+    "        inputs = self.tokenizer(\n",
+    "            text,\n",
+    "            padding=True,\n",
+    "            truncation=True,\n",
+    "            max_length=280,\n",
+    "            return_tensors='pt',\n",
+    "        ).to(self.device)\n",
+    "        output = self.model(**inputs)\n",
+    "        prediction = output[0].softmax(1)\n",
+    "        tensors = prediction.detach().cpu().numpy()\n",
+    "        result = np.argmax(tensors)\n",
+    "        confidence = tensors[0][result]\n",
+    "        return f\"Rank: {result}, {100 * confidence:.2f}%\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": [
+       "<__main__.FrankenBert at 0x1ffa9d15550>"
+      ]
+     },
+     "metadata": {},
+     "execution_count": 5
+    }
+   ],
+   "source": [
+    "FrankenBert('saved_model')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = FrankenBert('saved_model')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": [
+       "'Rank: 0, 98.96%'"
+      ]
+     },
+     "metadata": {},
+     "execution_count": 8
+    }
+   ],
+   "source": [
+    "model.predict(\"Mickey Mouse is in the house\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "isText = model.predict(\"Mickey Mouse is in the house\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": [
+       "str"
+      ]
+     },
+     "metadata": {},
+     "execution_count": 11
+    }
+   ],
+   "source": [
+    "type(isText)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ]
+}
```

Comment:
I've made a note to delete this file when we clean our data at the end of the project. I don't think this notebook will help any future teams work on this project.